types/backbone/index.d.ts

```diff
@@ -293,34 +293,32 @@ declare namespace Backbone {
          * persisted to the server. Unset attributes will be set to undefined.
          * You can also pass an attributes object to diff against the model,
          * determining if there *would be* a change. */
-        changedAttributes(attributes?: any): any;
-        clear(options?: Silenceable): any;
+        changedAttributes(attributes?: Partial<T>): Partial<T> | false;
+        clear(options?: Silenceable): this;
         clone(): Model;
-        destroy(options?: ModelDestroyOptions): any;
-        escape(attribute: string): string;
-        has(attribute: string): boolean;
-        hasChanged(attribute?: string): boolean;
+        destroy(options?: ModelDestroyOptions): JQueryXHR | false;
+        escape(attribute: keyof T & string): string;
+        has(attribute: keyof T & string): boolean;
+        hasChanged(attribute?: keyof T & string): boolean;
         isNew(): boolean;
         isValid(options?:any): boolean;
-        previous(attribute: string): any;
-        previousAttributes(): any;
-        save(attributes?: any, options?: ModelSaveOptions): any;
-        unset(attribute: string, options?: Silenceable): Model;
-        validate(attributes: any, options?: any): any;
-
-        private _validate(attributes: any, options: any): boolean;
+        previous<a extends keyof T & string>(attribute: a): T[a]| null | undefined;
+        previousAttributes(): Partial<T>;
+        save(attributes?: Partial<T> | null, options?: ModelSaveOptions): JQueryXHR;
+        unset(attribute: keyof T & string, options?: Silenceable): this;
+        validate(attributes: Partial<T>, options?: any): any;private _validate(attributes: any, options: any): boolean;
```

Comment:
This was probably an accidental deletion.

```suggestion
        validate(attributes: Partial<T>, options?: any): any;
        
        private _validate(attributes: any, options: any): boolean;
```